src/main/resources/openapi/NotificationsAPIs.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Notifications APIs
-  version: '1.16.3'
+  version: '1.16.5'
```

Comment:
nbd jumped a version number. Not that it matters, but since this is meant to be semantic versioning, I think it should be 1.17.0

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
>MAJOR version when you make incompatible API changes
>MINOR version when you add functionality in a backward compatible manner
>PATCH version when you make backward compatible bug fixes